app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -208,10 +224,48 @@ class PipelineSampleReport extends React.Component {
       search_taxon_id: searchTaxonId,
       thresholded_taxons,
       selected_taxons,
+      selected_taxons_top: selected_taxons.slice(0,  this.max_rows_to_render),
+      pagesRendered: 1,
+      rendering: true,
       rows_passing_filters: selected_taxons.length
+    },
+    () => {
+      this.setState({rendering: false})
+    }
+    );
+  }
+
+  //Load more samples on scroll
+  scrollDown() {
+    var that = this;
+    $(window).scroll(function() {
+      if ($(window).scrollTop() > $(document).height() - $(window).height() - 6000) {
```

Comment:
6000?

Reply:
Does this mean you've scrolled past >6000 invisible items?

Reply:
6000 pixels. basically every time you scroll ;-)